packages/open-ui/src/interfaces/IntentOption.ts

```diff
@@ -0,0 +1,6 @@
+export type IntentOption =
```

Comment:
`IntentOption` --> `IntentOptions` ?

Reply:
@remotealex just thinking loud here, is `Primary` or `Secondary` falls into Intent ? Like what I understood is Intent is any action. `Primary` or `Secondary` I am associating with a brand ? Whatcha think ?

Reply:
Singular `IntentOption` makes sense to me in context: 
```
interface ButtonProps {
   intent?: IntentOption;
   ...
```

If we split primary&secondary into its own prop, seperate from `intent`, we'll have to do weird manual overrides. Keeping them in one place makes sense from an implementation perspective. Maybe we can change it from intent to another keyword?